one-ui/src/main/java/com/marklogic/hub/oneui/controllers/ModelController.java

```diff
@@ -60,4 +62,18 @@ ModelManager modelManager() {
         JsonNode json = modelManager.getModel(modelName);
         return new ResponseEntity<>(json, HttpStatus.OK);
     }
+
+    @RequestMapping(value = "/job-info", method = RequestMethod.GET)
+    @ResponseBody
+    public ResponseEntity<?> getLatestJobInfoForAllModels() {
+        List<JsonNode> modelList = modelManager.getLatestJobInfoForAllModels();
+        return new ResponseEntity<>(modelList, HttpStatus.OK);
```

Comment:
One tweak I'd make because I like single-line methods in a controller - no need to change now, but I'd just do this:

    return new ResponseEntity<>(modelManager.getLatestJobInfoForAllModels(), HttpStatus.OK);

And since I'm really lazy, I'd have an AbstractController class with generic reusable methods, such as one that would let me do this here instead:

    return ok(modelManager.getLatestJobInfoForAllModels());

Reply:
Good idea.

Reply:
Added https://github.com/marklogic/marklogic-data-hub/pull/3695/commits/3029f5529211221da4ad83bd1e2a2403857c99fd